File: king-admin-java/src/main/java/com/oukingtim/ServletInitializer.java
package com.oukingtim;

import org.springframework.boot.builder.SpringApplicationBuilder;
import org.springframework.boot.web.support.SpringBootServletInitializer;

/**
 * Created by oukingtim
 */
public class ServletInitializer extends SpringBootServletInitializer {
    @Override
    protected SpringApplicationBuilder configure(SpringApplicationBuilder builder) {
        return builder.sources(KingAdminJavaApplication.class);
    }
}

File: king-admin-kotlin/src/main/kotlin/com/oukingtim/ServletInitializer.kt
package com.oukingtim

import org.springframework.boot.builder.SpringApplicationBuilder
import org.springframework.boot.web.support.SpringBootServletInitializer

/**
 * Created by oukingtim
 */
class ServletInitializer : SpringBootServletInitializer() {
    override fun configure(builder: SpringApplicationBuilder): SpringApplicationBuilder {
        return builder.sources(KingAdminKotlinApplication::class.java!!)
    }
}